release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -144,6 +144,77 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-storage"]
 === Storage
 
+==== CSI volume health (Technology Preview)
+Container Storage Interface (CSI) volume health is currently enabled, and supported as a Technology Preview feature. However, {product-title} currently does not ship any CSI drivers that support this feature.
+
+==== CSI driver volume detach with non-graceful node shutdown (Technology Preview)
+Container Storage Interface (CSI) drivers can now automatically detach volumes when a node goes down non-gracefully. When a non-graceful node shutdown occurs, you can manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is supported with Technology Preview status.
+
+For more information, see ???.
+
+==== VMware vSphere encryption support is generally available
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on vSphere. This feature allows user-provisioned persistent storage to be encrypted over wire.
+
+For more information, see ???.
+
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters as well.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage classes defined as the default, thus temporarily having two default storage classes, before removing default status from the previous default storage class.
```

Comment:
s/second storage classes/second storage class

Consider splitting this second sentence into two. 

> This feature makes it easier to change the default storage class because you can create a second storage classes defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class.